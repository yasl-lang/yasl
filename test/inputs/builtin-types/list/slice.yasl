const x = [1, 2, 3, 4, 5]

echo x[0:5]
echo x[1:3]
echo x[0:-1]
echo x[-10:20]

let letters = [.a, .b, .c, .d, .e, .f]

echo letters[:3]
echo letters[3:]
echo letters[:]
