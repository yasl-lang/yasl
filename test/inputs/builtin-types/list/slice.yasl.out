[1, 2, 3, 4, 5]
[2, 3]
[1, 2, 3, 4]
[1, 2, 3, 4, 5]
[a, b, c]
[d, e, f]
[a, b, c, d, e, f]
